rustls/src/webpki/verifier_builders.rs

```diff
@@ -159,7 +159,7 @@ impl ClientCertVerifierBuilder {
 /// instead of constructing a builder.
 #[derive(Debug, Clone)]
 #[non_exhaustive]
-pub enum ClientCertVerifierBuilderError {
+pub enum CertVerifierBuilderError {
```

Comment:
Naming nit: how about tightening this up to `VerifierBuilderError`? I feel like the `Cert` prefix is fairly superfluous in this context?

Reply:
SGTM. Done.